src/components/section-list.component.js

```diff
@@ -16,43 +17,45 @@ type Props = {
   buttonAction: Function,
 };
 
+const Section = styled.View`
+  margin-top: 15px;
+`;
+const SectionTitle = styled.Text`
+  color: ${colors.black};
+  padding: 15px;
+  ${fonts.fontPrimaryBold};
+`;
+const StyledTitleView = styled.View`
+  padding: 15px;
+`;
+const LoadingIcon = styled(ActivityIndicator)`
+  margin: 20px 0px;
```

Comment:
Of course, shorthands properties need units, but except for `0`. So here can be changed as `margin: 20px 0;`.